src/mame/drivers/spg2xx_vii.cpp

```diff
@@ -1,7 +1,7 @@
 // license:BSD-3-Clause
 // copyright-holders:Ryan Holtz, David Haywood
 
-/* 'Zone' systems */
+/* Siatronics/KenSingTon/JungleSoft Vii */
```

Comment:
Please don’t make random changes like this.  You’re likely to cause a conflict considering this file is being touched by @DavidHaywood pretty frequently.

Reply:
Reverted.